analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt

```diff
@@ -30,6 +30,23 @@ import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
 
 class MavenSupportTest : WordSpec({
+    "getOriginalScm()" should {
+        "return the parent's SCM if the child SCM uses the parent's SCM URL implicitly" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    url = "https://github.com/spring-projects/spring-boot/childArtifactName"
+                }
+                parent = MavenProject().apply {
+                    scm = Scm().apply {
+                        url = "https://github.com/spring-projects/spring-boot"
+                    }
+                }
+            }
+
+            MavenSupport.getOriginalScm(mavenProject)?.url shouldBe "https://github.com/spring-projects/spring-boot"
+        }
```

Comment:
Could you please add a similar test for the connection as well? Or update this test to test both?